docs/README.md

```diff
@@ -130,10 +205,32 @@ pipeline:
 ...
 ```
 
+ALternatively, you can use a plugin for log monitoring. This `config.yaml` collects logs from MySQL via a plugin and sends them to Elasticsearch. By default, MySQL plugin collects general, slow query, and error logs but can be configured to collect MariaDB Audit logs as well by adding `enable_mariadb_audit_log: true` to the config file. More details of the MySQL plugin can be viewed [here](https://github.com/observIQ/stanza-plugins/blob/master/plugins/mysql.yaml).
```

Comment:
Not a fan of calling out Elasticsearch if we have not tested it recently.

Reply:
replaced elasticsearch examples